src/piidigger/filehandlers/docx.py

```diff
@@ -9,10 +9,9 @@
 from docx2python.iterators import iter_paragraphs
 
 from piidigger.filehandlers._sharedfuncs import ContentHandler
-from piidigger.globalvars import (

```

Comment:
Broke these apart for better readability.  Repeated other places where the paranthesized list approach was used.